SwiftAudio/Classes/AudioPlayer.swift

```diff
@@ -257,6 +260,19 @@ public class AudioPlayer: AVPlayerWrapperDelegate {
         }
     }
     
+    func setPitchAlgorithm(forItem item: AudioItem) {
```

Comment:
Use the static members in [AVAudioTimePitchAlgorithm](https://developer.apple.com/documentation/avfoundation/avaudiotimepitchalgorithm).

Reply:
I believe that's only supported from iOS 11+, would you want to make the minimum OS of this library iOS 11?

Reply:
Trying to make these changes now but i keep getting an error where it detects AVAudioTimePitchAlgorithm as an `NSString *` and can't find method `varispeed`.. wonder if our configs are different somehow.

I'm on Xcode 10, which are you on?

Reply:
That is weird 🤔  I'm on Xcode 10 too (Version 10.0 (10A255)). I can pull your branch again and test.

Reply:
Ok, let me know. Also trying to figure out what's going on my side. I've got all the other changes ready.

Reply:
Tested out and it all works fine on Xcode 10, iOS 12. Even tested on Xcode 9 and iOS 10.3.1 and no problems, so I am not sure what is wrong